spec/Powertrain/Battery.vspec

```diff
@@ -376,6 +382,12 @@ Charging.TimeToComplete:
            E.g. if charging shall start in 3 hours and 2 hours of charging is needed,
            then Charging.TimeToComplete shall report 5 hours.
 
+Charging.Displayed:
+  type: uint32
+  datatype: float
+  unit: s
+  description: The time displayed to the customer which is needed for the current charging process to reach complete charge.
```

Comment:
How will this one differ from Charging.TimeToComplete?

Is it remaining time or time stamp it is supposed to continue?

Reply:
TimeToComplete is related to ChargeLimit. And the one I have proposed is always related to full charge of current net capacity. I will add comment here.